submissions/goldOchim/aboutMe/HTML/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
This file is inside the wrong folder. The `HTML` folder shouldn't be inside `aboutMe` folder. instead, it should be inside your name folder

Reply:
Are you having an issue updating this change?